lib/data_feeds/solis_cloud_api.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'base64'
+
+module DataFeeds
+  class SolisCloudApi
+    class ApiFailure < StandardError; end
+    class NotFound < StandardError; end
+    class NotAllowed < StandardError; end
+    class NotAuthorised < StandardError; end
+
+    BASE_URL = 'https://www.soliscloud.com:13333'
+
+    def initialize(api_id, api_secret)
+      @api_id = api_id
+      @api_secret = api_secret
+    end
+
+    def user_station_list
+      get_data('/v1/api/userStationList', {})
+    end
+
+    def station_day(id, day)
+      sleep 2
```

Comment:
This might avoid throttling for this specific instance, but if we've got multiple reloads happening (e.g. triggered by admins) and daily loads across servers then we might still hit throttle. So we should have some kind of retry behaviour in the requests handling too. We're using Faraday Retry elsewhere.

Reply:
added retry and limiter